rel-eng/build-packages-for-obs.sh

```diff
@@ -140,6 +140,26 @@ while read PKG_NAME PKG_VER PKG_DIR; do
   ${VERBOSE:+cat "$T_LOG"}
 
   eval $(awk '/^Wrote:.*src.rpm/{srpm=$2}/^Wrote:.*.changes/{changes=$2}END{ printf "SRPM=\"%s\"\n",srpm; printf "CHANGES=\"%s\"\n",changes; }' "$T_LOG")
+  EXTRA_CHANGELOGS=0
+  if [ "$(head -n1 ${CHANGES}|grep '^- ')" == "" ]; then
+    PREVIOUS_CHANGELOGS=0
+  else
+    PREVIOUS_CHANGELOGS=1
+  fi
+  FILELIST="$(ls ${GIT_DIR}/${PKG_DIR}${PKG_NAME}.changes.* 2> /dev/null; true)"
+  for FILE in $FILELIST; do
+    if [ ${PREVIOUS_CHANGELOGS} -eq 0 -a ${EXTRA_CHANGELOGS} -eq 0 ]; then
+      sed -i '1i\\' ${CHANGES}
+    fi
+    EXTRA_CHANGELOGS=1
+    LINENUMBER=1
+    while IFS= read -r LINE; do
+      if [ "${LINE}" != "" ]; then
```

Comment:
Some users add empty lines by mistake. The changelog checker @cbbayburt is doing should be handling this, but for now we had this to avoid an error in the script.